data/config-schema/src/main/resources/net/corda/schema/configuration/rbac.1.0/corda.rbac.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "https://corda.r3.com/net/corda/schema/configuration/rbac/1.0/corda.rbac.json",
+  "title": "Corda RBAC Configuration Schema",
+  "description": "Configuration schema for the Roll Based Access section.",
+  "type": "object",
+  "default": {},
+  "properties": {
+    "password": {
+      "description": "Settings for passwords.",
+      "type": "object",
+      "default": {},
+      "userPasswordChangeExpiry": {
+        "description": "The amount of time (days) before the password must be updated again after user password change.",
+        "type": "integer",
+        "minimum": 30,
+        "maximum": 1095,
+        "default": 90
+      },
+      "adminPasswordChangeExpiry": {
+        "description": "The amount of time (days) before the password must be updated again after admin password change",
+        "type": "integer",
+        "minimum": 1,
+        "maximum": 30,
+        "default": 7
+      },
+      "passwordExpiryWarningWindow": {
+        "description": "The time (days) before a password expires in which we begin to offer warnings about upcoming expiry.",
+        "type": "integer",
+        "minimum": 1,
+        "default": 30,
+        "maximum": 365
+      }
```

Comment:
We might need some input from @jamesbr3 with regards to best practice, but I thought that these days the best practice was not to enforce password expiry but instead focus on complexity (which we may want to make a rule that can be changed by configuration).

Do we even need an expiry policy? Shouldn't Corda be un-opinionated about this and instead offer the ability to report on password age so that integrators can integrate with their own policy.

Either way, I think don't think forcing a maximum is correct as this would prevent a customer setting their own policy.

Reply:
Thanks guys, I removed the maximum values from these configs.